tools/README.md

```diff
@@ -0,0 +1,75 @@
+# AnDOtools
```

Comment:
Hi @Slowblitz 
Overall, I wouldn't organize this as a separate project... So I think it would be better to have the "doc" for the tools in the main README.md file, and not to have a tools/README.md file like the present one... Can you move its content (or what's relevant) to the main AnDOChecker README.md file?

Reply:
Done merging and correcting all readme file in the main readme